config/optional/views.view.og_members_overview.yml

```diff
@@ -25,7 +25,7 @@ display:
       access:
         type: perm
         options:
-          perm: 'access content'
```

Comment:
Is this a bad merge?

Some of these changes to og.services.yml and og.module may be coming from a merge.

Reply:
I believe so, see https://github.com/Gizra/og/pull/476.